packages/web-ui/src/sections/Document/Editor/Playground/index.tsx

```diff
@@ -0,0 +1,106 @@
+'use client'
+
+import { useCallback, useEffect, useMemo, useState } from 'react'
+
+import { ConversationMetadata } from '@latitude-data/compiler'
+import { Badge, Button, Input, Text } from '$ui/ds/atoms'
+
+import { Header } from '../Header'
+import Chat from './Chat'
+import Preview from './Preview'
+
+function convertParams(
+  inputs: Record<string, string>,
+): Record<string, unknown> {
+  return Object.fromEntries(
+    Object.entries(inputs).map(([key, value]) => {
+      try {
+        value = JSON.parse(value)
+      } catch (e) {
+        // Do nothing
+      }
+      return [key, value]
+    }),
+  )
+}
+
+export default function Playground({
+  metadata,
+}: {
+  metadata: ConversationMetadata | undefined
+}) {
+  const [mode, setMode] = useState<'preview' | 'chat'>('preview')
```

Comment:
Playground has two modes: `preview` and `chat`. This is just internal and **only** used in this component, so there is no need of making it an enum... right?

Reply:
@geclos ☝️ another example of poor developers confused with enums por culpa de Ferran